js/forms.js

```diff
@@ -279,85 +279,181 @@
+        var matchStart = $el.text().toLowerCase().indexOf("" + string.toLowerCase() + ""),
+        matchEnd = matchStart + string.length - 1,
+        beforeMatch = $el.text().slice(0, matchStart),
+        matchText = $el.text().slice(matchStart, matchEnd + 1),
+        afterMatch = $el.text().slice(matchEnd + 1);
+        $el.html("<span>" + beforeMatch + "<span class='highlight'>" + matchText + "</span>" + afterMatch + "</span>");
+        if (img.length) {
+          $el.prepend(img);
+        }
+      }
+
+      // function on keyUp
+      function keyUp(e, data, $input, $autocomplete) {
+
+        // select element on Enter
+        if (e.which === 13) {
+          $autocomplete.find('li')[currentLi].click();
+          return;
+        }
+
+        // hide list on ESC button
+        if (e.which === 27) {
+          $input.blur();
+        }
+
+        // prevent refreshing list on pressing ctrl, shif, alt, arrow, f1-f12, etc
+        // helpful if we selected something using arrow and we press ctrl for example
+        if(
+        [9, 16, 17, 18, 19, 20, 33, 34, 35, 36, 37, 38, 39, 40, 45, 112, 113, 114, 115, 116, 117, 118, 119, 120, 121, 122, 123].indexOf(e.which) < 0
+        ){
+          highlightElements(data, $input, $autocomplete);
+        }
+      }
+
+      // function to displaying elements
+      function highlightElements(data, $input, $autocomplete){
+        // Check if the input isn't empty and isn't equal to the old option
+        if (val !== '' && currentAutocompleteVal !== val) {
+          var val = $input.val().toLowerCase(),
+          maxElementsAmount = options.maxElementsAmount;
+          $autocomplete.empty();
+
+          for (var key in data) {
+
+            if (maxElementsAmount === 0) {
+              break;
+            }
+
+            if (data.hasOwnProperty(key) &&
+            key.toLowerCase().indexOf(val) !== -1 &&
+            key.toLowerCase() !== val) {
+              if (maxElementsAmount !== null) {
+                maxElementsAmount--;
+              }
+
+              autocompleteOption = $('<li></li>');
+              if (!!data[key]) {
+                autocompleteOption.append('<img src="' + data[key] + '" class="right circle"><span>' + key + '</span>');
+              } else {
+                autocompleteOption.append('<span>' + key + '</span>');
+              }
+              $autocomplete.append(autocompleteOption);
+
+              highlight(val, autocompleteOption);
+            }
+          }
 
-        // Check if data isn't empty
-        if (!$.isEmptyObject(data)) {
-          // Create autocomplete element
+          currentAutocompleteVal = val;
+          autocompleteOption = $autocomplete.find('li');
+          currentLi = 0;
+          $(autocompleteOption[currentLi]).addClass('active');
+        }
+      }
+
+      // arrow usage - on key down
+      function arrowUsage(e, $autocomplete) {
+        // Capture up and down key
+        if (
```

Comment:
I changed that, but all programmers has own style and there isn't "the style" which you can use in each situation. It's about preferences. If we want to have the same style in each file we should configure task in "gulp watch" which will be reformating code after save. That's the only way to have the same style in the whole project if you have more than 2 programmers ;)